packages/aws-cdk/lib/toolkit/cli-io-host.ts

```diff
@@ -21,7 +21,21 @@ interface IoMessage {
   readonly action: IoAction;
 
   /**
-   * A short code uniquely identifying message type.
+   * A short message code uniquely identifying a message type of the format [CATEGORY]_[NUMBER_CODE].
+   * All 'error' level message codes begin with 0.
+   * All warning level message codes begin with 1.
+   * All info level message codes begin with 2.
```

Comment:
Personally I think it'd be nicer to use letters for this: `E`, `W`, `I`. But that's a bit of bikeshedding.